app/controllers/Challenge.scala

```diff
@@ -136,20 +136,27 @@ final class Challenge(
   def decline(id: String) =
     Auth { implicit ctx => _ =>
       OptionFuResult(api byId id) { c =>
-        if (isForMe(c)) api decline c
+        if (isForMe(c)) api decline(c, None)
```

Comment:
Unsure on purpose of this `decline` method and if `reason` should be implemented here.